src/v2/api/index.md

```diff
@@ -1426,7 +1426,7 @@ type: api
 
 - **Details:**
 
-  Contains parent-scope `v-on` event listeners (without `.native` modifiers). This can be passed down to an inner component via `v-on="$listeners"` - useful when creating higher-order components.
+  Contains parent-scope `v-on` event listeners (without `.native` modifiers). This can be passed down to an inner component via `v-on="$listeners"` - useful when creating transparent wrapper components.
```

Comment:
I think we've talked about this before, are we all ok with transparent wrapper instead of higher-order? I think higher-order is more commonly used so it's possible it will be understood much faster by people learning, but I could go both ways on it.

Reply:
Isn't this different from an HOC? The definition [used by React](https://reactjs.org/docs/higher-order-components.html) is:

> a function that takes a component and returns a new component

This is a component rather than a function and it returns raw elements rather than a new component.

Reply:
Not necessarily, from the same document just a paragraph lower:

> a higher-order component transforms a component into another component.

Having worked with both, I'm not sure I'd make such a distinction, but if you think that it's helpful for understanding, I'm all for it.

Reply:
Sounds good!